<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>700</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>setup</string>
    </property>
   </widget>
   <widget class="QLabel" name="test_label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>260</y>
      <width>451</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="killEM">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>710</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="accessibleDescription">
     <string>Kliknij aby zabić świat</string>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>do końca</string>
    </property>
    <property name="shortcut">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="QLabel" name="MAPKA">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1500</width>
      <height>850</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>740</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>x fal</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>480</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>lata</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_to_the_end">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>740</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>data końca</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_okrazenia">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>701</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="specialValueText">
     <string>L. okrążeń</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
   </widget>
   <zorder>MAPKA</zorder>
   <zorder>start</zorder>
   <zorder>test_label</zorder>
   <zorder>killEM</zorder>
   <zorder>pushButton</zorder>
   <zorder>label</zorder>
   <zorder>push_to_the_end</zorder>
   <zorder>spinBox_okrazenia</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
